### ISSUE TITLE
Support latest lambda aws runtime

### ISSUE THREAD

**Issue by dmitriy-pisarevskiy** [aws-cloudformation/cfn-lint#2256]
hey! aws already released new version runtime for labmda "dotnet6"
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-runtime

![image](https://user-images.githubusercontent.com/70516019/165302700-1f620623-04cd-45f1-9cc0-096c6bbe0214.png)

cfn-lint --version
cfn-lint 0.56.0

**Comment by kddejong** [comment]
We are pulling this value back from the boto packages we should have it shortly.
```
"LambdaRuntime": {
   "botocore": "lambda/2015-03-31/Runtime"
},
```

**Comment by kddejong** [comment]
This was included as of `v0.58.2` do an update of cfn-lint
